src/views/IncorporationAgreement.vue

```diff
@@ -0,0 +1,316 @@
+            </div>
+            <div class="download-link-container">
+              <span>
+                <v-icon color="blue">mdi-file-pdf-outline</v-icon>
+                <a href="/files/benefit_company__corporation_agreement.pdf" download>
+                  Download the Sample Incorporation Agreement and Benefit Company Articles
+                </a>
+              </span>
+            </div>
+          </div>
+        </v-card>
+      </div>
+    </section>
+    <section class="mt-10">
+      <header id="share-structure-header">
+        <h2>3. Confirm Incorporation Agreement and Article Completion</h2>
+      </header>
+      <AgreementType />
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+import { AgreementType } from '@/components/IncorporationAgreement'
+
+@Component({
+  components: {
+    AgreementType
+  }
+})
+export default class IncorporationAgreement extends Vue {
+  private helpToggle: boolean = false
+  private readMoreFlag: boolean = false
+
+  private get documentURL (): string {
+    return 'https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/' +
+    'business-management/permits-licences-and-registration/registries-forms/' +
+    'benefit_company__corporation_agreement.pdf'
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+ul {
+  list-style-type: disc;
+}
+
+li {
+  padding-top:0.25rem
+}
+
+.articles-statements{
+  margin-left: 1.3rem;
+}
+
+.articles-statements-footer {
+  margin-left: 1.8rem;
+  margin-top: 1.2rem
+}
+
+p{
+  padding-top: 0.5rem;
+}
+
+a {
+  text-decoration: none;
+  color: #1976d2;
+}
+
+.help-btn {
+  cursor: pointer;
+  color: #1976d2;
+  vertical-align:middle;
+}
+
+.read-more-btn {
+  cursor: pointer;
+  color: #1976d2;
+}
+
+.incorporation-agreement-help {
+  margin: 2rem 0;
+  border-top: 1px dashed $gray6;
+  border-bottom: 1px dashed $gray6;
+  padding: 1rem 0;
+
+  h2, h4 {
+    padding-top: 1rem;
+  }
+
+  u {
+    display: flex;
+    direction: rtl;
+  }
+}
+
+.article-stmt-icon {
+  margin-right: 0.2rem;
+}
+
+.template-details {
+  display: flex;
+  justify-content: center;
+  margin-top: 0.5rem;
+}
+
+.preview-download-container {
+  display: flex;
+  justify-content: center;
+  margin-top: 1.5rem;
+  align-items: center;
+}
+
+.download-link-container {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  margin-left: 1rem;
+  border-top: solid 1px #DEE2E6;
+  border-bottom: solid 1px #DEE2E6;
+}
+
+.share-structure-check-panel {
+  padding: 2rem;
+  vertical-align: middle;
+}
+
+.share-structure-check-text {
+  font-size: 0.875rem;
+  padding-left: 0.2rem
+}
+
+.share-structure-check-header {
+  font-size: 0.875rem;
+  font-weight: bold;
+  padding-bottom: 0.5rem;
+}
+
+.preview-image {
+  width: 200px;
+  height: 259px;
+}
+
+.tool-tip {
```

Comment:
Are the other tooltips in this project supposed to have this style too?

Reply:
(If yes then it makes sense to move this CSS to base.scss.)

Reply:
I am not sure about it. I see that people and roles has a similar styling

Reply:
I think Scott's intent is for all tooltips to appear as dotted-underline text. That's why I think the tooltip in ListPeopleAndRoles should be updated.

Reply:
There the tool tip is for a person/org with a very long name in the table. I dont think we should be using a dotted line there.

Reply:
Oh, OK. I agree with you.